docs/gds-supported-platform/accessing-concourse.md

```diff
@@ -1,174 +1,28 @@
-endpoint with `fly`.
-
-## Setting up the tunnel
-
-```sh
-kubectl port-forward service/gsp-concourse-web -n gsp-system 8080:8080
-```
-
-You should be able to visit [`http://127.0.0.1:8080/`](http://127.0.0.1:8080/)
-and see the user interface.
-
-The username and password by default are `admin:password`. Based on the need to
-setup the tunnel, we believe these do not need to be changed.
+GSP provides [Concourse](http://ci.local.govsandbox.uk) for all your building, testing and deploying needs. You
+can interact with it through a browser or with `fly`.
 
 ## Using `fly`
 
-We strongly discourage the use of `fly`. The Build and Run team are working
-hard to provide a consistent way of auto-deploying and updating the pipelines
-with a GitOps solution.
-
-The above means, any manual change to the CI system with `fly`, will be
-overwritten by an operator.
-
-You may find a need of going against the recommendation, for instance you'd like
-to `hijack` a container to debug a failed `job`.
-
-Concourse on the landing page, should encourage you to download the binary used
-to interact with its API.
-
-- [macOS](http://127.0.0.1:8080/api/v1/cli?arch=amd64&platform=darwin)
-- [Windows](http://127.0.0.1:8080/api/v1/cli?arch=amd64&platform=windows)
-- [GNU/Linux](http://127.0.0.1:8080/api/v1/cli?arch=amd64&platform=linux)
-
-Once downloaded you will be able to login and manage your pipelines.
-
-In order to login run:
-
-```sh
-fly -t cluster.staging login -c http://127.0.0.1:8080/
-```
-
-The above command will ask you to visit a link in your web browser. This will
-authenticate you without providing any login credentials into the CLI.
-
-To push your first pipeline, run:
-
-```sh
-fly -t cluster.staging set-pipeline -c ci/build-prototype.yaml -p build-prototype
-```
-
-For more information on `fly`, seek the
-[documentation](https://concourse-ci.org/fly.html).
-
-## Overcoming default configuration
-
-Sometimes you may wish to allow public access to your Concourse. You can
-achieve this by modifying the `values.yaml` file for your cluster and blending
-the following into existing setup:
-
-```yaml
-concourse:
-  concourse:
-    web:
-      externalUrl: https://concourse.example.com
-  web:
-    ingress:
-      enabled: true
-      annotations:
-        kubernetes.io/tls-acme: "true"
-        kubernetes.io/ingress.class: nginx
-      hosts:
-      - concourse.example.com
-      tls:
-      - hosts:
-        - concourse.example.com
-        secretName: concourse-tls-cert
-```
-
-This will establish the URL Concourse would be accessible under, as well as
-setup the Let's Encrypt to provide a SSL certificate.
-
-Do remember, the default username and password are easily obtainable. You may
-wish to change these too. This can be achieved by adding the following to the
-`values.yaml` file:
+You can interact with Concourse using [`fly`](https://concourse-ci.org/fly.html). You must make sure the version of `fly` matches the version of Concourse that is running.
```

Comment:
Maybe add a few lines on how to sync this?

When I run `fly -t main login -c http://ci.local.govsandbox.uk/`, I get this warning:
```
WARNING:

fly version (4.2.3) is out of sync with the target (5.3.0). to sync up, run the following:

    fly -t main sync
```
However, running `fly -t main sync`, I get this:
```
error: unknown target: main
```

I've also tried this (which produces the same error):

```
fly -t gsp-local sync
```

Reply:
You can upgrade `fly` manually—if you browse to `http://ci.local.govsandbox.uk/` there's a bunch of links for each platform in the bottom right.